pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -586,3 +587,26 @@ var _ = Describe("ImageUpload", func() {
 		)
 	})
 })
+
+type atomicBool struct {
```

Comment:
I am a bit confused here.
Changing or accessing a boolean is usually atomic by default, same for `int` set/get.

Did you meant to use the same mutex between different values?

Reply:
I used atomic because the package which provides go-routine save methods for various types uses the same term. It is less about atomic operation, it is about synchronizing memory between go routines.